protos/dcs/common/v0/common.proto

```diff
@@ -201,22 +201,17 @@ message Unit {
   Coalition coalition = 4;
   // The DCS type-name of the unit. e.g "MiG-29A", "ZSU_57_2" or "Hawk ln"
   string type = 5;
-  // The position of the unit
-  Position position = 6;
+  // Information about the unit in 3D space.
+  Transform transform = 6;
```

Comment:
I am still unsure if `transform` is a good name, but still don't have a better idea.

Reply:
Works for me.

Reply:
@omwdunkley called it _spatial_ [here](https://github.com/DCS-gRPC/rust-server/issues/159#issuecomment-1170409986), so maybe also a good candidate for a name? As a non-native English speaker I don't know which fits better. I'll keep `transform` unless someone else jumps in promoting a different name.

Reply:
I think transform works fine.